packages/cli/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "src",
-    "outDir": "lib"
-  }
+    "outDir": "./lib"
+  },
+  "include": [
+    "./src"
+  ]
```

Comment:
Thanks for cleaning up.
Maybe we could open an issue to maybe add `baseUrl` and `paths` to get rid of the `../../../../../` in most places?
A quick blog for reference [here](https://decembersoft.com/posts/say-goodbye-to-relative-paths-in-typescript-imports/)

Reply:
That might be a good idea. 😄